Assign2/main.py

```diff
@@ -280,7 +283,7 @@ def experiment(injection_tubing_diameter, num_wells, num_connections, mass_flow_
 
 
 # TO DO: Add in the DOE generation functionality. 
-imported_df = pd.read_csv("./file_import_and_graph/DOE_tanner.csv", index_col=0)
+imported_df = pd.read_csv("C:/Users/Beerstein/GitHub/MDO_Project/Assign2/file_import_and_graph/DOE_tanner.csv", index_col=0)
```

Comment:
can you revert this change back to the original? This might cause others code to fail. "./file_import_and_graph/DOE_tanner.csv"
```suggestion
imported_df = pd.read_csv("./file_import_and_graph/DOE_tanner.csv", index_col=0)
```

Reply:
support tanner's suggestion.

Reply:
Fixed